MagiskOnWSA/installer/Install.ps1

```diff
@@ -53,10 +57,9 @@ Function Finish {
     Start-Process "wsa://com.android.vending"
 }
 
-If (Test-CommandExist pwsh.exe) {
+If (Check-Windows11 -And Test-CommandExist pwsh.exe) {
```

Comment:
Use
``` powershell
If ((Check-Windows11) -And (Test-CommandExist 'pwsh.exe')) {
```

Reply:
I have tested, and my PR one works, but using brackets has better readability, so I have updated.

Reply:
Thank you.